Content.Server/BluespaceHarvester/BluespaceHarvesterRiftSystem.cs

```diff
@@ -0,0 +1,70 @@
+using Robust.Shared.Random;
+
+namespace Content.Server.BluespaceHarvester;
+
+public sealed class BluespaceHarvesterRiftSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        var query = EntityQueryEnumerator<BluespaceHarvesterRiftComponent, TransformComponent>();
+        while (query.MoveNext(out var uid, out var comp, out var xform))
+        {
+            comp.PassiveSpawnAccumulator += frameTime;
```

Comment:
The last couple of times sloth has said that it can cause different behavior on devices with different performance. Just, it's worth making sure there are no problems with 2 things here:

1) Setting the game to pause, and tracking the time in that state
2) Behavior on devices with different frame rates.